src/chrome/content/rules/Pando.xml

```diff
@@ -7,7 +7,7 @@
 		- pandoblog.com                (dm1.fastdomain.com)
 		- www.pandoblog.com            (dm1.fastdomain.com)
 -->
-<ruleset name="Pando (bad certs)" default_off="Wrong hostnames in certs.">
```

Comment:
Actually, I think it's totally reasonable to use a rough regex and then go through after and fix up issues on a one-off basis. Just suggesting that the test can help find some of the issues.